README.rst

```diff
@@ -0,0 +1,260 @@
+VSI          2014    `Visual Saliency-induced Index <https://ieeexplore.ieee.org/document/6873260>`_
+-            2016    `Content Score <https://arxiv.org/abs/1508.06576>`_
+-            2016    `Style Score <https://arxiv.org/abs/1508.06576>`_
+HaarPSI      2016    `Haar Perceptual Similarity Index <https://arxiv.org/abs/1607.06140>`_
+MDSI         2016    `Mean Deviation Similarity Index <https://arxiv.org/abs/1608.07433>`_
+MS-GMSD      2017    `Multi-Scale Gradient Magnitude Similiarity Deviation <https://ieeexplore.ieee.org/document/7952357>`_
+LPIPS        2018    `Learned Perceptual Image Patch Similarity <https://arxiv.org/abs/1801.03924>`_
+PieAPP       2018    `Perceptual Image-Error Assessment through Pairwise Preference <https://arxiv.org/abs/1806.02067>`_
+DISTS        2020    `Deep Image Structure and Texture Similarity <https://arxiv.org/abs/2004.07728>`_
+===========  ======  ==========
+
+No Reference
+^^^^^^^^^^^^
+
+===========  ======  ==========
+Acronym      Year    Metric
+===========  ======  ==========
+TV           1937    `Total Variation <https://en.wikipedia.org/wiki/Total_variation>`_
+BRISQUE      2012    `Blind/Referenceless Image Spatial Quality Evaluator <https://ieeexplore.ieee.org/document/6272356>`_
+===========  ======  ==========
+
+Feature based
+^^^^^^^^^^^^^
+
+===========  ======  ==========
+Acronym      Year    Metric
+===========  ======  ==========
+IS           2016    `Inception Score <https://arxiv.org/abs/1606.03498>`_
+FID          2017    `Frechet Inception Distance <https://arxiv.org/abs/1706.08500>`_
+GS           2018    `Geometry Score <https://arxiv.org/abs/1802.02664>`_
+KID          2018    `Kernel Inception Distance <https://arxiv.org/abs/1801.01401>`_
+MSID         2019    `Multi-Scale Intrinsic Distance <https://arxiv.org/abs/1905.11141>`_
+===========  ======  ==========
+
+Benchmark
+---------
+
+As part of our library we provide code to benchmark all metrics on a set of common Mean Opinon Scores databases.
+Currently only `TID2013`_ and `KADID10k`_ are supported. 
+You need to download them separately and provide path to images as an argument to the script.
+
+Here is an example how to evaluate SSIM and MS-SSIM metrics on TID2013 dataset:
+
+.. code-block:: bash
+
+   python3 tests/results_benchmark.py --dataset tid2013 --metrics SSIM MS-SSIM --path ~/datasets/tid2013 --batch_size 16
+
+We report `Spearman's Rank Correlation cCoefficient <https://en.wikipedia.org/wiki/Spearman%27s_rank_correlation_coefficient>`_ (SRCC) 
+and `Kendall rank correlation coefficient <https://en.wikipedia.org/wiki/Kendall_rank_correlation_coefficient>`_ (KRCC). 
+We do not report `Pearson linear correlation coefficient <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`_ (PLCC) 
+as it's highly dependent on fitting method and is biased towards simple examples.
+
+For metrics that can take greyscale or colour images, ``c`` means chromatic version.
+
+.. csv-table:: Benchmark results
```

Comment:
The default style of the table title looks sad. Let's use separate heading or no heading at all.